source/service_test.go

```diff
@@ -56,6 +57,18 @@ func TestEndpoints(t *testing.T) {
 			},
 		},
 		{
+			"annotated services return an endpoint",
```

Comment:
can we distinguish the test from the one before?

e.g.

`"annotated services return an endpoint with target IP"` and
`"annotated services return an endpoint with target hostname"`